web/src/components/tasks/BusinessStructureTask.test.tsx

```diff
@@ -192,4 +192,16 @@ describe("<BusinessStructureTask />", () => {
       expect(currentUserData().taskProgress[taskId]).toEqual("COMPLETED");
     });
   });
+
+  it("updates operating phase from GUEST_MODE to GUEST_MODE_WITH_BUSINESS_STRUCTURE when business structure is saved", async () => {
```

Comment:
I'd love a corresponding test here that it does NOT update to GUEST_MODE_WITH_BUSINESS_STRUCTURE when the user is not in GUEST_MODE

Right now, if I deleted the ternary in the OnboardingLegalStructure and just had it update to GUEST_MODE_WITH_BUSINESS_STRUCTURE always, then this test would still pass and nothing would fail